doc/pm-example-float.json

```diff
@@ -22,7 +22,7 @@
     "CodeMeaning": "Mono-exponential diffusion model"
   },
   "SourceImageDiffusionBValues": ["0","1400"],
-  "AnatomicRegionCode": {
+  "AnatomicRegionSequence": {
```

Comment:
I think this one should be `AnatomicRegionCodeSequence`, if we want to be consistent.